fedoragsearch-transforms/islandora_transforms/slurp_all_MODS_to_solr.xslt

```diff
@@ -57,6 +57,17 @@
       </xsl:otherwise>
         </xsl:choose>
     </field>
+
+    <-- JIRA TRAC-876 Define utk_mods_etd_author_orcid in Solr -->
```

Comment:
Looks like there's some invalid syntax here; comments should be `<!-- $text -->`

Reply:
My bad.

Reply:
Deleted.